python/simulation.cpp

```diff
@@ -6,24 +8,174 @@
 
 #include "context.hpp"
 #include "error.hpp"
+#include "pyarb.hpp"
 #include "recipe.hpp"
+#include "schedule.hpp"
+
+namespace py = pybind11;
 
 namespace pyarb {
 
-void register_simulation(pybind11::module& m) {
+enum class py_spike_recording {
```

Comment:
A comment documenting what this is for would be helpful.

Reply:
I did think the code in `py_simulation.record()` was pretty self-explanatory.

Reply:
I meant, a quick comment that explains what the `py_spike_recording` enum is to be used for?